doc/Type/Any.pod6

```diff
@@ -203,7 +203,7 @@ the C<Seq> will be iterating over its elements, calling C<.take> on them.
 
 Defined as:
 
-    method deepmap(&block --> List) is nodal
+    method deepmap(Any: &block, *%_ --> List) is nodal
```

Comment:
In this case, for instance, it does not make any sense to add *%. The code is simply:

https://github.com/rakudo/rakudo/blob/8134470538457c68164eb64f90e8d3e946d235a0/src/core.c/Any.pm6#L168

this means that that argument is going to be discarded. As said in my comment, we *really* need to think about this on a method-by-method basis.